--- a/arch/arm/boot/dts/overlays/Makefile
+++ b/arch/arm/boot/dts/overlays/Makefile
@@ -184,7 +184,11 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	miniuart-bt.dtbo \
 	mipi-dbi-spi.dtbo \
 	mlx90640.dtbo \
+	mm610x-spi.dtbo \
+	mm610x-spi-pi5.dtbo \
+	mm810x-spi.dtbo \
+	mm_wlan.dtbo \
 	mmc.dtbo \
 	mz61581.dtbo \
 	ov2311.dtbo \
 	ov5647.dtbo \
@@ -228,6 +232,7 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	qca7000-uart0.dtbo \
 	ramoops.dtbo \
 	ramoops-pi4.dtbo \
+	raven.dtbo \
 	rootmaster.dtbo \
 	rotary-encoder.dtbo \
 	rpi-backlight.dtbo \
